ere-guests/empty-program/zisk/Cargo.toml

```diff
@@ -6,7 +6,7 @@ rust-version.workspace = true
 license.workspace = true
 
 [dependencies]
-ziskos = { git = "https://github.com/0xPolygonHermez/zisk.git", rev = "f9a3655" }
+ziskos = { git = "https://github.com/0xPolygonHermez/zisk.git" }
```

Comment:
This will be patched to `v0.11.0` as well

Reply:
If ZisK default branch one day moves to v0.12.0 and Cargo.lock is updated, then the patch in the other file won't be applied anymore, no?
Wouldn't be safer to keep the `tag="v0.11.0"` here?